research-hub-web/src/app/components/home/browse/browse.component.scss

```diff
@@ -1 +1,50 @@
-@import "../../../../styles";
\ No newline at end of file
+@import '../../../../styles.scss';
+@import '../../../../themes.scss';
+@import '../../../../typography.scss';
```

Comment:
What happened to import aliases?

Reply:
oh yeah jest was disabling it for some reason but it should be fine to use import alias again